### PR TITLE
fix(card): add "locked" svg image and centered image option

### PR DESCRIPTION
**Pull request by ranst91-u**
Preview: https://static-staging.usabilla.com/stylabilla/pr/blup-199-chart-type/docs/section-components.html

## What to test? Scope of the test?

Added a "locked" svg image in the thumbnail card options. This will be used for "Coming soon" chart types

## Reviews

Reviewers: @chiel @MLKiiwy @kagnamazian @realdreamer @ranst91-u 

I assume the rest of the checks are not needed, since it's just a small fix

### CODE REVIEW

File: src/components/_card.scss [hunk 1]
```diff
@@ -172,7 +172,7 @@ $sb-thumbnail-desc-color: $sb-color-light-1;
 $sb-thumbnail-title-color: $sb-color-dark-2;
 $sb-thumbnail-title-active-color: $sb-color-blue-2;
 
-$sb-thumbnail-image-list: mood nps opentext responses rating;
+$sb-thumbnail-image-list: mood nps opentext responses rating locked;
```
Comment (realdreamer) [inline src/components/_card.scss]:
```suggestion
$sb-thumbnail-image-list: mood nps opentext responses rating lock;
```
or `coming-soon` something like

Comment (chiel) [reply src/components/_card.scss]: When it comes to iconography, I would just pick a name that represents what the icon is - the context in which it is used should only matter to naming if it is named within that context.

Comment (ranst91-u) [reply src/components/_card.scss]: so "lock" or "coming soon"?

Comment (MLKiiwy) [reply src/components/_card.scss]: lock 👍

**Pull request merged**